services/ui-src/src/App.js

```diff
@@ -35,8 +36,36 @@ function App() {
 
     userHasAuthenticated(false);
 
-    history.push("/login");
+    history.push("/");
+  }
+
+  async function handleLogin(event) {
+    event.preventDefault();
+    try {
+      const localLogin = config.LOCAL_LOGIN === "true";
+      if (localLogin) {
+        const alice = {
+          username: "alice",
+          attributes: {
+            given_name: "Alice",
+            family_name: "Foo",
+            email: "alice@example.com",
+          },
+        };
+        loginLocalUser(alice);
+        userHasAuthenticated(true);
```

Comment:
This is going to be a brittle point. I don't see an obvious alternative, maybe a comment suggesting that it could be removed to harden the system if desired (assuming they were past the point where rapid local dev was necessary).

Reply:
Agreed.  We've been discussing removing the local auth mock.  We actually discussed doing it in this issue, but figured it warranted its own.
FWIW, we see our options as:
1. keep the mock
2. use [local stack](https://github.com/localstack/localstack) pro, which mocks cognito
3. change the local dev workflow to fetch a remote cognito installation's info, probably master branch by default, but with the option to point to any... this is solid, and its something we do today, but its not the standard workflow.  it also would make "fetching aws creds" as part of the standard dev workflow, which is fine by me, but we should think it through
4. change the local dev workflow to actually deploy the individual dev's cognito service to amazon, and hook into that.  This has all the implications of number 3.  The benefit is seen as more freedom for the dev.  If you want to bootstrap users with different attributes than whats on master, you can do that without fear of collision.

Agree with your comment here.  I'm going to hold short of creating an issue, only because I'm confident this won't fall through the cracks.  It's actively being discussed.

Reply:
Tagging you in case it's helpful to hear our current thinking.  Its definitley something still up in the air.  
@whytheplatypus